core/client/app/mixins/ed-editor-api.js

```diff
@@ -0,0 +1,101 @@
+import Ember from 'ember';
+
+var EditorAPI = Ember.Mixin.create({
+    getValue: function () {
+        return this.get('element').value;
```

Comment:
Just code-style-wise, this threw me for a loop for a bit. I think our convention's been to work with `this.$()` like you did in `getSelection`, rather than going for the raw dom element